src/d3a/models/myco_matcher/external_matcher.py

```diff
@@ -7,14 +7,13 @@
 
 import d3a.constants
 from d3a.d3a_core.exceptions import (
-    InvalidBidOfferPairException, OfferNotFoundException,
-    BidNotFoundException, MycoValidationException)
+    InvalidBidOfferPairException, MycoValidationException)
 from d3a.d3a_core.redis_connections.redis_area_market_communicator import ResettableCommunicator
 from d3a.models.market import Market
 
 
 class ExternalMatcherEventsEnum(Enum):
-    OFFERS_BIDS_RESPONSE = "offers_bids_response"
+    MATCHING_DATA_RESPONSE = "matching_data_response"
```

Comment:
Can you please explain the reason for this renaming? AFAIS, there are still only lists of bids and offers sent. The new name suggests to me that there was already some matching done. Is this because you also return the `filters`? I understand that these filters were sent by the myco and only returned here convenience, right?

Reply:
The matching algorithms should receive bids/offers + atm some of them receive an additional member which is a timestamp, so in order for this to be scalable I renamed the variable, we could send even more data in the future

Reply:
`some of them receive an additional member which is a timestamp` Can you please explain which timestamp and why only some of them have this member?
Even if some of them contain a timestamp, this is still not a good rationale for renaming this IMO.

Reply:
The pay as clear matching algorithm expects the matching data to be {"market": {"bids": [], "offers":[], "current_time: ""}}
So the very outer key is the market uuid with mapping to bids/offers and some additional data
so the matching_data makes more sense to me than bids_offers, especially after taking into consideration that this can be scalable and we could be sending more data in the future

Reply:
@hannesdiedrich let me know what you think, if this is okay I will continue with the renaming and create the Myco PR

Reply:
Hmm, I am still not convinces but would like to hear other opinions here. If more people agree with you, please use your suggestion.

Reply:
I have decided to leave this change to my next feature to minimize the number of changes that should be tested